dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -2482,5 +2500,160 @@ public void putRelationshipWithJson() throws Exception {
+                                                                       isParentOrgUnitOf.getID().toString())
+                                                                .contentType(MediaType.parseMediaType
+                                                                    (org.springframework.data.rest.webmvc.RestMediaTypes
+                                                                         .TEXT_URI_LIST_VALUE))
+                                                                .content(
+                                                                    "https://localhost:8080/server/api/core/items/" + orgUnit1
+                                                                        .getID() + "\n" +
+                                                                        "https://localhost:8080/server/api/core/items" +
+                                                                        "/" + orgUnit2
+                                                                        .getID()))
+                                                   .andExpect(status().isCreated())
+                                                   .andReturn();
+
+
+        itemService.getMetadata(orgUnit1, "*", "*", "*", "*", true);
+
+        getClient(adminToken).perform(get("/api/core/items/" + orgUnit1.getID()))
+                             .andExpect(status().isOk())
+                             .andExpect(jsonPath("$.metadata['relation.isParentOrgUnitOf'][0].value",
+                                                 is(String.valueOf(orgUnit2.getID()))));
+        getClient(adminToken).perform(get("/api/core/items/" + orgUnit2.getID()))
+                             .andExpect(status().isOk())
+                             .andExpect(jsonPath("$.metadata['relation.isChildOrgUnitOf'][0].value",
+                                                 is(String.valueOf(orgUnit1.getID()))));
+
+
+
+    }
+
+    @Test
+    public void orgUnitFindByLabelParentChildOfCountTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        EntityType orgUnit = entityTypeService.findByEntityType(context, "OrgUnit");
+        RelationshipType isParentOrgUnitOf = relationshipTypeService
+            .findbyTypesAndTypeName(context, orgUnit, orgUnit, "isParentOrgUnitOf", "isChildOrgUnitOf");
+
+        MetadataSchema metadataSchema = metadataSchemaService.find(context, "relation");
+        MetadataFieldBuilder.createMetadataField(context, metadataSchema, "isParentOrgUnitOf", null, null).build();
+        MetadataFieldBuilder.createMetadataField(context, metadataSchema, "isChildOrgUnitOf", null, null).build();
+
+        String adminToken = getAuthToken(admin.getEmail(), password);
+
+        context.restoreAuthSystemState();
+        // Here we create our first Relationship to the Publication to give it a dc.contributor.author virtual
+        // metadata field.
+        getClient(adminToken).perform(post("/api/core/relationships")
```

Comment:
Again, `post` should cleanup after itself

Reply:
This has been updated